### DISCUSSION TITLE
Dont close search (find files) window clicking on "Go to file" button

### DISCUSSION THREAD

**Discussion by Akrotkiv** [doublecmd/doublecmd#1505] [Suggestions]
Please don't close search (find files) window after clicking on "go to file" button.
Also features like sorting results and copy/cut/delete would be nice in results box.

I was accustomed to use "Altap Salamander" file explorer and like their search window features.

**Comment by JonnyTech** [comment]
If you go back to the search dialog, the results remain.

For sorting, etc, click feed to listbox then you manage the files.

@alexx2000 is the word "listbox" too technical for end users? Maybe file panel instead?

**Reply by Akrotkiv** [reply]
OK. thanks. I see that works ;)
